source/documentation/subscribe-ern.html.md.erb

```diff
@@ -0,0 +1,35 @@
+---
+title: Subscribe an ERN to push/pull notifications
```

Comment:
I feel like this should be 'Subscribe an ERN to receive push/pull notifications' Maybe?